net/gemget/Portfile

```
@@ -0,0 +1,85 @@
+                        size    102899 \
+                    github.com/spf13/pflag \
+                        lock    v1.0.5 \
+                        rmd160  2ce81608a38c6f383a35bccd24d64361df5828c9 \
+                        sha256  7f41acdcba65b1fab5b9b633947a139f9915b60f94bdab486cdbe9d90c54f61e \
+                        size    50815 \
+                    github.com/schollz/progressbar \
+                        lock    v3.6.0 \
+                        rmd160  f29fc1fcf32f76726e46b136ffdd805b895681e5 \
+                        sha256  35cfdfbfbe29162f42d5dea127774a22c14967133f99992f5e023df2f1f3918d \
+                        size    601968 \
+                    github.com/pmezard/go-difflib \
+                        lock    v1.0.0 \
+                        rmd160  fc879bfbdef9e3ff50844def58404e2b5a613ab8 \
+                        sha256  7cd492737641847266115f3060489a67f63581e521a8ec51efbc280c33fc991f \
+                        size    11409 \
+                    github.com/mitchellh/colorstring \
+                        lock    d06e56a500db \
+                        rmd160  79e1fb92818b77a56b274c3bb7880891af3f7829 \
+                        sha256  0a3c9097c65cf50b9dfe8150adf2f096f9e62b36200759459969d3b9ee3a20fe \
+                        size    4679 \
+                    github.com/mattn/go-runewidth \
+                        lock    v0.0.9 \
+                        rmd160  412c0e508e55f4fe437be0f71d7d22eca2b4366f \
+                        sha256  4f0f4a22257ccecfb6beae88052d850380ecc0e806d6bcc92d3656ebcac3b638 \
+                        size    16716 \
+                    github.com/makeworld-the-better-one/go-gemini \
+                        lock    v0.11.0 \
+                        rmd160  8b52426f5ed2718c7e1d637de4e2dd2da0053dc3 \
+                        sha256  297368c8e482401448132cc6b04b41b87ff213111119861653aa7cd77f0e4391 \
+                        size    11863 \
+                    github.com/google/go-cmp \
+                        lock    v0.3.1 \
+                        rmd160  66e42f672a5a40561c388b78b3644abd926e7bef \
+                        sha256  86ee7c90714e7eb5c60d1a8a515235daef806df454c767043b593540e958167f \
+                        size    76416 \
+                    github.com/dustin/go-humanize \
+                        lock    v1.0.0 \
+                        rmd160  e8641035159ea3e8839ee086f01f966443956303 \
+                        sha256  e45e3181c07b3e2dad8e1317e91a5bbbee4845067e3e3879a585a5254bc6a334 \
+                        size    17273 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.1 \
+                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
+                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
+                        size    42171
+
+set go_ldflags      "-s -w -X main.version=${version}"
+build.args          -ldflags \"${go_ldflags}\"
```

```suggestion
build.args          -ldflags '-X main.Version=${version}'
```

Haven't tested this, but it should work.